roborazzi/src/main/java/com/github/takahirom/roborazzi/RobolectricDeviceQualifiers.kt

```diff
@@ -0,0 +1,153 @@
+package com.github.takahirom.roborazzi
+
+object RobolectricDeviceQualifiers {
+  // Generated by [GenerateQualifiersTest.kt]
+  // Data from: AOSP https://android.googlesource.com/platform/tools/base/+/mirror-goog-studio-master-dev/sdklib/src/main/java/com/android/sdklib/devices
+// FROM: tv.xml
+  const val AndroidTV1080pLand =
+    "w7670dp-h4314dp-xlarge-long-notround-land-television-xhdpi-keyshidden-dpad"
+  const val AndroidTV1080pPort =
+    "w7670dp-h4314dp-xlarge-long-notround-port-television-xhdpi-keyshidden-dpad"
+  const val AndroidTV720pLand =
+    "w7670dp-h4314dp-xlarge-long-notround-land-television-tvdpi-keyshidden-dpad"
+  const val AndroidTV720pPort =
+    "w7670dp-h4314dp-xlarge-long-notround-port-television-tvdpi-keyshidden-dpad"
+
+  // FROM: devices.xml
+  const val Device135FreeformLand =
+    "w1706dp-h960dp-xlarge-long-notround-land-any-hdpi-keyshidden-nonav"
+  const val Device135FreeformPort =
+    "w1706dp-h960dp-xlarge-long-notround-port-any-hdpi-keyshidden-nonav"
+  const val Device76FoldinwithouterdisplayPort =
```

Comment:
My personal take: Device76 (or the like) tells me absolutely nothing.


I find it very confusing. The names should be device related.
Nobody knows what device135 is.

I'd rather do not try to be too clever downloading the devices from a file to generate this file but set them directly, like Paparazzi.

Otherwise you are risking to make breaking changes in the future, plus, you can easily make the names easy to read and understand, what is indeed the point of the feature imo